app/views/components/errorSummary.scala.html

```diff
@@ -0,0 +1,36 @@
+@*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.govukfrontend.views.html.components._
+
+@this(govukErrorSummary: GovukErrorSummary)
+
+@(errors: Seq[FormError], errorFieldName: Option[String] = None)(implicit messages: Messages)
+
+@errorList = @{errors.map { error =>
+    ErrorLink(
+        href = Some(s"#${errorFieldName.getOrElse(error.key)}"),
+        content = Text(messages(error.message, error.args: _*))
+    )
+}}
+
+@if(errorList.nonEmpty) {
+    @govukErrorSummary(
+        ErrorSummary(
+            errorList = errorList,
+            title = Text(messages("error.summary.title"))
+        ))
```

Comment:
Just style, close the brackets in column that you open it, more readable

So last bracket to the new line. Style comment